src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -190,15 +189,13 @@ object ErrorAggregator {
 
     val stats = statsSchema.fieldNames.map(_.toLowerCase)
     val sumCols = stats.map(s => sum(s).as(s))
-    val countCols = clientCounts.flatMap{
+    val countCols = clientCounts map {
```

Comment:
I thought we were going to remove all `HLL` cols except `client_count`?

Reply:
Right you are!